src/libraries/System.Formats.Tar/src/System/Formats/Tar/TarHeader.Write.cs

```diff
@@ -893,32 +893,22 @@ private string GenerateExtendedAttributeName()
         }
 
         // Gets the special name for the 'name' field in a global extended attribute entry.
-        // Format: "%d/GlobalHead.%p/%n"
+        // Format: "%d/GlobalHead.%p.%n"
         // - %d: The path of the $TMPDIR variable, if found. Otherwise, the value is '/tmp'.
         // - %p: The current process ID.
-        // - %n: The sequence number of the global extended header record of the archive, starting at 1. In our case, since we only generate one, the value is always 1.
```

Comment:
Thanks for removing. Remnant from before we approved the class to describe PAX GEA entries.